docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1g.md

```diff
@@ -0,0 +1,48 @@
+# does_zone_meet_G3_1_1g
+**Schema Version:** 0.0.22  
+
+**Description:** returns a boolean value indicating whether the zone meets G3_1_1g:
+ 
+**part1:** "If the baseline HVAC system serves HVAC zones that includes computer rooms,  baseline system 11 shall be used where the baseline HVAC system type is 7 or 8 and the total computer room peak cooling load is greater than 600,000 BTU/h (175 kW) AND <= 3,000,000"
+
+**part 2:** If the baseline HVAC system serves HVAC zones that includes computer rooms,  Baseline System 11 shall be used for such HVAC zones in buildings with a total computer room peak cooling load greater than 3,000,000 Btu/h.
+
+**part 3:** If the baseline HVAC system serves HVAC zones that includes computer rooms,  baseline system 3 or 4 shall be used for all HVAC other zones - that is zones where the computer room peak cooling load is= <600,000 Btu/h, and when the peak cooling load is between 600,000 btu/hr and 3,000,000 btu/hr and the baseline system type is not system 7 or 8.
+
+**Inputs:** 
+- **B_RMI**
+- **zone_id**
+- **starting_system_type**
+
+**Returns:**  
+- **result**: an enum - either NO or G1, G2, G3 for the 3 parts
+
+**Function Call:**
+- **get_component_by_id**
+- **get_zones_computer_rooms**
+
+## Logic:
+- set the result variable to NO - only a positive test can give it a different value: `result = NO`
+- get the list of zones with computer rooms using the function get_zones_computer_rooms to get a list of zones including computer rooms: `computer_room_zones_dict = get_zones_computer_rooms(B_RMI)`
+- if the zone_id is in the computer_room_zones_dict, then this is a computer room zone and it is eligible: `if zone_id in computer_room_zones_dict:`
+    - create a variable for the total peak computer room load: `total_computer_peak_cooling_W = 0`
+    - loop through the computer room zones: `for computer_room_zone_id in computer_room_zones_dict:`
+    - get the zone: `computer_room_zone = get_component_by_id(computer_room_zone_id)`
+    - loop through the spaces in the zone: `for space in computer_room_zone:`
+        - check if the space is a computer room zone by checking the space lighting type: `if space.lighting_space_type in ["COMPUTER_ROOM"]:`
+            - loop through the miscellaneous loads in the space: `for load in space.miscellaneous_equipment:`
+                - check if the load is an information technology type: `if load.miscellaneous_equipment_type == "INFORMATION_TECHNOLOGY_EQUIPMENT":`
+                    - we assume that the load is a computer load and add it to the `total_computer_peak_cooling_W`: `total_computer_peak_cooling_W += load.power`
```

Comment:
You are calculating computer peak load, but App G language refers to  "computer room peak cooling load", so I think we need to include all equipment, lighting and occupancy loads. 
Aside from that, I recommend not isolating computer room spaces but instead accounting for peak load of computer room zones. Then, you can just call get_zone_peak_internal_load function here for all zones returned by get_zones_computer_rooms. This is aligned with the update to that language in 90.1 2022 - see G3.2.1.2 (d)

If others disagree, then you would have to do the following:
(a) apply the logic that we used in get_zone_peak_internal_load function to computer room spaces, and 
(b) apply logic that we used in get_zones_computer_rooms to identify spaces that are computer rooms withing the zones that are returned by this function. E.g., I think we had some criteria there in addition to space.lighting.space_type.

Reply:
thanks, Maria.  I updated based on your comments.  We can discuss on the call next week.